app/layouts/default-layout.tsx

```diff
@@ -197,6 +198,9 @@ export default createFragmentContainer(DefaultLayout, {
     fragment defaultLayout_session on JwtToken {
       ciipUserBySub {
         __typename
+        firstName
+        lastName
+        emailAddress
```

Comment:
This will generate a warning, as Relay expects values to be used in the same file as the query.